tests/test_find_in_range.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+from finders.find_in_range import find_stops_in_range
+from finders.gps import GPSPosition
+from finders.stops import Stop
+
+SAMPLE_STOPS = [
+    Stop("location_sw", GPSPosition(-10, -10))
+    , Stop("location_se", GPSPosition(10, -10))
+    , Stop("location_ne", GPSPosition(10, 10))
+    , Stop("location_nw", GPSPosition(-10, 10))
+]
+
+
+TEST_DATA = [
+    (GPSPosition(1, 1), Stop("location_ne", GPSPosition(10, 10)))
```

Comment:
In your test data, you are supposed to have the same number of values as indicated in @pytest.mark.parametrize
So since you have current_gps_location, range_value and expected_stops
You supposed to have three elements per line, the one you seem to be missing is the range_value